keywording/text.xml

```diff
@@ -421,14 +421,53 @@ dependency for your package).
 </p>
 
 <p>
-Some architectures (<c>alpha</c>, <c>ia64</c>, <c>mips</c>, <c>riscv</c>,
+Some architectures (<c>alpha</c>, <c>loong</c>, <c>mips</c>, <c>riscv</c>,
 <c>s390</c>) do not maintain a stable keyword, so packages are not to be marked
 stable for one of these architectures.
 </p>
 
 </body>
 </subsection>
 
+<subsection>
+<title>Stabilization groups</title>
+<body>
+
+<p>
+Stabilization groups are sets of packages defined to help organize and automate
+stabilization requests. These groups are stored under
+<c>metadata/stabilization-groups/</c>, and all files within this directory
+(including subdirectories) are included recursively.
+</p>
+
+<p>
+Each group is named according to its path, for example <c>@gnome/vte</c>. This
```

Comment:
```suggestion
Each group is named after its path, for example <c>@gnome/vte</c>. This
```

Reply:
Waiting for ACK from anyone else (I'm also far from native speaker)

Reply:
Either are OK but sbraz's suggestion sounds better.